packages/block-library/src/heading/index.js

```diff
@@ -31,6 +32,30 @@ const supports = {
 export const settings = {
 	title: __( 'Heading' ),
 	description: __( 'Introduce new sections and organize content to help visitors (and search engines) understand the structure of your content.' ),
+	getAccessibilityLabel( attributes ) {
```

Comment:
This is concerning as it isn't part of the public API for block registration. It would be used only by the mobile app only, right?

Reply:
This I think could be a good/easy way for 3rd parties to add a simple descriptive accessibility label to the blocks that will be read by mobile screen readers.

It would be awesome if web could also use it, if it doesn't, then yes, it's only for mobile apps.

Reply:
You need to keep in mind that block registration is heavily used by 3rd party developers. That's why every change proposed goes through RFC (Request For Comments) process. You can check https://github.com/WordPress/gutenberg/pull/13693 for reference. 

One way of providing this functionality is to temporary use "unstable" prefix as described here:

https://github.com/WordPress/gutenberg/blob/e6fc9c4df1168a27bb5a933d61e30afe0c1710c7/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis

Reply:
Thank you for the explanation!

It looks like the `__experimental` prefix will fit well. We can implement it on mobile, and if it proves to be a good enough, we can do a proper RFC.

Does this sound good?
cc @Tug

Reply:
Sorry for the confusion, I didn't intend to have it merged as it is, my intent was merely to experiment on an alternative to #15214 which passes a function into the block attributes. I don't currently have a solution for this, but I'm open to suggestions.

A prefix could work yeah if that's what gutenberg devs recommend.
A RFC could be another approach if we think it's useful for the web as well.

At the time I wrote this I was thinking about either using the `blocks.registerBlockType` hook to add our custom property or splitting index.js into a native and web version but those aren't very good.